### PR TITLE
Fetched data from DB to More Info Component

### PR DESCRIPTION
**Pull request by ferrary7**
- Fetched data from my MongoDB to the more info Component.
- Styling not done for this commit.
- Minor Bug fixes.

### CODE REVIEW

**Comment by PatelNeelMahesh** [comment]
Click on edit and add PR description.
image

File: client/.env [hunk 1]
```diff
@@ -1,2 +1 @@
-REACT_APP_SERVER_URL=http://localhost:3000
-REACT_APP_API_URL=https://ll.thespacedevs.com/2.2.0/launch/upcoming/?limit=100&format=json&offset=0
\ No newline at end of file
+REACT_APP_SERVER_URL=http://localhost:3000
```
Comment (PatelNeelMahesh) [inline client/.env:1]: Add `.env` in `.gitignore` and remove it from github.

File: client/src/App.js [hunk 2]
```diff
@@ -13,6 +14,7 @@ function App() {
       <Route path="/users/signup" element={<SignUp />} />
```
Comment (PatelNeelMahesh) [inline client/src/App.js:14]: Add `path='*'` and create a `404` component.

**Review by PatelNeelMahesh — COMMENTED** [review]
After doing the required changes you can merge pull request.

**Review by PatelNeelMahesh — CHANGES_REQUESTED** [review]
Implement the above asked changes, before adding more features.

File: client/.env (hunk 1 above)
Comment (ferrary7) [reply client/.env:1]: This will be taken care in the very next commit for sure.

File: client/src/App.js (hunk 2 above)
Comment (ferrary7) [reply client/src/App.js:14]: This will be taken care in the very next commit for sure.

**Pull request merged**